docs/guide-for-local-setup/README.md

```diff
@@ -0,0 +1,40 @@
+# This is a document that describes how to run RDS app locally with local backend
+
+## 1. Setup the Backend
+
+- Go to the RDS Backend setup guide [here](https://github.com/Real-Dev-Squad/website-backend/#readme)
+- Add the `cors` config in `config/local.js` file:
+
+```javascript
+ cors: {
+     allowedOrigins: /(http:\/\/localhost:[0-9]{4}$)/,
+ },
+```
+
+## 2. Setup the Frontend -
+
+- Go to the README of the respective frontend repo.
+- Change all the client_id to **your client id**.
+  - RDS client id: `23c78f66ab7964e5ef97`
+- Change all the API endpoints to `http://<your local api>:<port>/`
+  - RDS API endpoint: `https://api.realdevsquad.com/`
+  - Default API endpoint[local]: `http://localhost:3000/`
+- Change all the url with respective frontend url
+  - Like you want to run WWW and My site then you have to set `http://localhost:5500` for WWW(main Site) then change all the url from `https://www.realdevsquad.com` to `http://localhost:5500` in both the repo and if you set `http://localhost:3443` for My site then you have to change all the url from `https://my.realdevsquad.com` to `http://localhost:3443` in both the repo and so on.
```

Comment:
We have a new route in backend `auth/github/login` If that is used, there is no need to add or change github clientID and all

Reply:
Done i remove the client ID change instruction